_posts/2024-03-20-strimzi-kraft-migration.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+title:  "Migrate your Strimzi-operated cluster from ZooKeeper to KRaft"
+date: 2024-03-20
+author: paolo_patierno
+---
+
+In the previous [blog post](https://strimzi.io/blog/2024/03/21/kraft-migration/) we explored how Apache ZooKeeper is used to store Apache Kafka clusters metadata and what are the current limitations.
+The Apache Kafka community introduced the new KRaft protocol which allows you to overcome these limitations and store metadata in Kafka itself, taking into account that the ZooKeeper support will be removed soon.
+For this reason, there is a need to migrate existing clusters from using ZooKeeper to KRaft for storing metadata.
+We also covered the overall Kafka metadata migration procedure.
+The migration procedure can be done without downtime, but it is not that straightforward because it requires manual configurations and several rolling updates.
+The good news is that if your cluster is powered by Strimzi, this process is much easier and can be accomplished simply by annotating your `Kafka` resource.
+In this blog post, we are going to show you how the Strimzi Cluster Operator provides a semi-automated process to run the migration easily, so that you don't need to worry about updating configurations and restarting cluster nodes.
+
+<!--more-->
+
+### Strimzi's migration support
+
+If you are using the Strimzi operator to run your current ZooKeeper-based cluster on Kubernetes, the migration process is semi-automated.
+All you need to do is update the `strimzi.io/kraft` annotation on the `Kafka` custom resource. 
```

Comment:
This makes it sound like you update it just once to `enabled`.  Maybe you can say something like ...
```suggestion
The migration process is managed by updating the `strimzi.io/kraft` annotation on the `Kafka` custom resource.
```